agent/cleanup.sh

```diff
@@ -29,6 +29,10 @@ if [[ "${AGENT_E2E_TEST_BOOT_MODE}" == "DISKIMAGE" ]]; then
     sudo podman rmi -f ${APPLIANCE_IMAGE} || true
 fi
 
+if [[ "${AGENT_E2E_TEST_BOOT_MODE}" == "ISO_NO_REGISTRY" ]]; then
+    sudo rm -rf "${OCP_DIR}/iso_builder"
```

Comment:
I found that `make clean` needs to be run twice because of this directory. The first time its run ocp/ostest isn't fully removed [here](https://github.com/openshift-metal3/dev-scripts/blob/master/ocp_cleanup.sh#L18) because its not run as sudo, so when subsequently running `make agent` we get a failure [here]( https://github.com/openshift-metal3/dev-scripts/blob/master/sanitychecks.sh#L10).

Probably should also remove ${OCP_DIR} here since it will still exist.

Reply:
nice. Thanks @bfournie

Reply:
You won't not get this issue if you set `AGENT_CLEANUP_ISO_BUILDER_CACHE_LOCAL_DEV` to `true`as it then removes the appliance cache and all other files except the generated OVE ISO to save the disk space.